hw/ip/aon_timer/data/aon_timer.hjson

```diff
@@ -265,6 +267,8 @@
           desc: "AON timer requested wakeup, write 0 to clear",
         }
       ]
+      tags: [// this could be updated by HW
+        "excl:CsrNonInitTests:CsrExclWriteCheck"],
```

Comment:
@tomroberts-lowrisc @tjaychen 
I feel this CSR can be updated by HW, so we may not be able to read back what SW writes. Please correct me if I'm wrong.

Reply:
@tomroberts-lowrisc 
can you take a look at these 2 exclusions? Thanks

Reply:
Confirmed those look correct - thanks @weicaiyang

Reply:
Thanks @tomroberts-lowrisc